### PR TITLE
Issue#1638

### LINKED ISSUE #1638: Delay type in api.md not equal real function
### Steps to reproduce

1) Show [delay type](https://github.com/redux-saga/redux-saga/tree/master/docs/api#delayms-val) effect
2) And compare with `delayP` function
https://github.com/redux-saga/redux-saga/blob/a1c465d18780bc69450569b76f43e4e35baac5c7/packages/delay-p/src/index.js#L3-L14
3) It is always return true

#### Description of the bug/issue
Where is the error in the documentation or in the implementation of the function `delayP` ?

#### Example
```js
const test = yield delay(1, "myValue"); // true
```
### Actual results
`test === true`

### The Expected results
```js
test === "myValue" // if the documentation is the source of truth
```

### Environment information
- redux-saga 1.0.0-beta.2
- babel

### PR DESCRIPTION
**Pull request by retyui**
| Q                        | A  |
| ------------------------ | ---  |
| Fixed Issues?            | Fixes #1638|
| Patch: Bug Fix?          |  Yes  |
| Major: Breaking Change?  |  No  |
| Minor: New Feature?      |   No  |
| Tests Added + Pass?      | Yes |
| Any Dependency Changes?  |  No  |

### CODE REVIEW

File: packages/delay-p/src/index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { CANCEL } from '@redux-saga/symbols'
 
-export default function delayP(ms) {
+export default function delayP(ms, returnValue = true) {
```
Comment (restrry) [inline packages/delay-p/src/index.js]: imho shouldn't have default value

Comment (shinima) [reply packages/delay-p/src/index.js]: I think the default value is fine. When we do not care the return value, we can omit the `returnValue`. And the default value (`true`) is useful when we use `delayP` in a race:

```javascript
const result= race({
  timeout: call(delayP, 2000),
  response: call(fetch, url),
})
// result.timeout is true if delayP resolves first
if (result.timeout) { /* ... */ }
else { /* ... */ }
```

Comment (shinima) [reply packages/delay-p/src/index.js]: The behavior now is the same as in v0.16.1. I think this is a small mistake when separating internal modules. 
https://github.com/redux-saga/redux-saga/blob/600802fe2c91be57c6d0c659424b9844739b5379/src/internal/utils.js#L94-L103

**Comment by retyui** [comment]
@restrry Is it ready to merge or do you have any question?

**Pull request merged**